django_kafka/consumer.py

```diff
@@ -29,25 +52,18 @@ class Consumer:
         https://docs.confluent.io/platform/current/clients/confluent-kafka-python/html/index.html#pythonclient-consumer
     """
 
-    topics: Topics[str, Topic]
+    topics: Topics
     config: dict
 
     polling_freq = settings.POLLING_FREQUENCY
     default_logger = logger
     default_error_handler = settings.ERROR_HANDLER
 
-    def __init__(self, config: Optional[dict] = None, **kwargs):
+    def __init__(self, **kwargs):
```

Comment:
Constructor level config override needed to be taken out;

1. We didn't use it anyway.
2. RetryConsumer _class_ needs to know the group ID of the main consumer, which is impossible to know _before_ Process forking since the consumer instance hasn't been created yet.

Reply:
yeah, we discussed that.